CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 * Update "location-units" version bumped to '2.0'. Refs UITEN-61.
 * Provide a link to session-locale settings from tenant-locale-settings. Refs UITEN-53.
 * Fix new addresses saving. Refs UIOR-459.
+* Update "stripes" to 'v3.0.0', "stripes-components" to '6.0.0', "stripes-core" to '4.0.0' and "react-intl" to '2.9.0'. Refs UITEN-83.
```

Comment:
This line should be [here](https://github.com/folio-org/ui-tenant-settings/pull/92/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR4).

Reply:
As I can see in `package.json` the current version is 2.14.0:
`"version": "2.14.0"`
So I put my comment in this block. Am I incorrect?

Reply:
Changes you made will be included in the next 2.15.0 release which is in progress.

Reply:
`2.14.0` was released in December so @VictorSoroka1 is correct. 

Typically, the first commit after a release should bump the package's minor version (e.g. from `2.14.0` to `2.15.0`) so that future work happens under that new, not-yet-released version. Sometimes we remember to do this immediately (e.g. [in stripes-components](https://github.com/folio-org/stripes-components/pull/1137)) but in less-active repositories, as here, sometimes we forget. Sorry!

Reply:
Got it, thanks guys!